### PR TITLE
Add ConfigService to horreum client

### PR DESCRIPTION
**Pull request by johnaohara**

### CODE REVIEW

File: horreum-client/src/main/java/io/hyperfoil/tools/HorreumClient.java [hunk 1]
```diff
@@ -1,19 +1,7 @@
...
-import io.hyperfoil.tools.horreum.api.services.DatasetService;
-import io.hyperfoil.tools.horreum.api.services.NotificationService;
-import io.hyperfoil.tools.horreum.api.services.ReportService;
-import io.hyperfoil.tools.horreum.api.services.RunService;
-import io.hyperfoil.tools.horreum.api.services.SchemaService;
-import io.hyperfoil.tools.horreum.api.services.SqlService;
-import io.hyperfoil.tools.horreum.api.services.SubscriptionService;
-import io.hyperfoil.tools.horreum.api.services.TestService;
-import io.hyperfoil.tools.horreum.api.services.UserService;
+import io.hyperfoil.tools.horreum.api.services.*;
```
Comment (jesperpedersen) [inline horreum-client/src/main/java/io/hyperfoil/tools/HorreumClient.java:4]: Lets keep the explicit imports

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

**Comment by jesperpedersen** [comment]
Merged to `master` - feel free to open a pull request for `0.7.x` if you need it.

Thanks for your contribution !

**Comment by johnaohara** [comment]
@jesperpedersen the tests were failing when this was merged. I just pushed another update to split out the ConfigService into interface and impl

**Comment by johnaohara** [comment]
I'll open another PR that fixes the test failures

**Pull request merged**